components/scripts/lib/gradle.sh

```diff
@@ -34,8 +34,11 @@ invoke_gradle() {
     -Dscan.capture-task-input-files=true
   )
 
-  # shellcheck disable=SC2206
-  args+=(${extra_args})
+  # https://stackoverflow.com/a/31485948
+  while IFS= read -r -d ''; do
+    args+=("$REPLY")
+  done < <( xargs printf '%s\0' <<< "$extra_args")
```

Comment:
Would there be any benefit in moving this block into a function in one of our library scripts (maybe config.sh)?

Reply:
Jim and I discussed this and we decided it is not worth the effort since working arrays with bash inside functions is not easy.